src/basic/calendarspec.c

```diff
@@ -420,7 +420,7 @@ static int parse_weekdays(const char **p, CalendarSpec *c) {
         assert(c);
 
         for (;;) {
-                unsigned i;
+                size_t i;
```

Comment:
I don't think we gain anything by changes like this. Overflow is not possible, so apart from making the hotpaths a bit slower, this has no effect.